tools/myst/config.c

```diff
@@ -69,6 +69,32 @@ static json_result_t _extract_mem_size(
     return ret;
 }
 
+static json_result_t _extract_start_address(
+    json_type_t type,
+    const json_union_t* un,
+    uint64_t* start_address)
+{
+    json_result_t ret = JSON_FAILED;
+    uint64_t value;
+
+    if (start_address == NULL)
+        CONFIG_RAISE(JSON_FAILED);
+
+    if (type == JSON_TYPE_INTEGER && un->integer > 0)
```

Comment:
The greater than zero check is redundant if check in line 84 is done correctly.

Reply:
Negative numbers should not be allowed as input and should be checked before the conversion to unsigned in line 84. At line 88, I am checking only for page alignment.